python/sglang/srt/managers/controller/infer_batch.py

```diff
@@ -9,6 +9,7 @@
 import torch
 from flashinfer.sampling import top_k_top_p_sampling_from_probs
 
+from sglang import global_config
```

Comment:
always use full path inside the package
```suggestion
from sglang.global_config import global_config
```